components/mdns/tests/host_test/components/esp_timer_linux/timer_task.hpp

```diff
@@ -16,7 +16,7 @@ typedef void (*cb_t)(void *arg);
 
 class TimerTaskMock {
 public:
-    TimerTaskMock(cb_t cb): cb(cb), t(run_static, this), active(false), ms(INT32_MAX) {}
+    TimerTaskMock(cb_t cb): cb(cb), active(false), ms(INT32_MAX) {}
```

Comment:
This class could make use of std::jthread.

Reply:
I would keep this goal for future, as using `jthread` needs C++20 and I had difficulties enabling in simply in CMake and supporting it in CI runner.